diesel/src/expression/expression_methods/sort_expression_methods.rs

```diff
@@ -0,0 +1,44 @@
+use expression::predicates::*;
+
+#[cfg(feature = "postgres")]
+pub trait Ordering : Sized {
```

Comment:
Is the name 'ordering' a good choice? It only set the ordering of nulls, right? Maybe call it `NullsOrdering`? (I don't have a preference.) Maybe also add a short doc comment.